instances/latest/terminologies/deviceType/wideBoreMRIScanner.jsonld

```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/deviceType/wideBoreMRIScanner",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/DeviceType",
+  "definition": "A 'wide-bore MRI scanner' is a closed high-field scanner which features a widened bore compared to the standard-bore MRI scanner. The typical magnetic strength lies between 1.0T and 3.0T.",
```

Comment:
delete field strength. this should be specified elsewhere

Reply:
done